pkgs/development/python-modules/beangulp/default.nix

```diff
@@ -0,0 +1,58 @@
+{
+  lib,
+  beancount,
+  buildPythonPackage,
+  chardet,
+  click,
+  fetchFromGitHub,
+  lxml,
+  petl,
+  python-magic,
+  pytestCheckHook,
+  regex,
+  setuptools,
+}:
+buildPythonPackage rec {
+  pname = "beangulp";
+  version = "0.2.0-unstable-2025-01-02";
```

Comment:
What's the issue with 0.2.0?

Reply:
None, didn't noticed that new release was tagged, updated package to use it.